### PR TITLE
v1.4.0 - package dependency updates

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- Upgraded to babel 7.
- Updated module dependencies.
- Travis uses node v8 & v10 to run builds.

### Fixed
- Travis build fixed by regenerating the yarn lock file.
- Lint parsing error fixed by adding "espree" resolution.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=6.0.0"
```
Comment (DamianMullins) [inline package.json]: Should probably bump this up to v8 if that's what we're trying the builds with.

**Comment by DamianMullins** [comment]
I think the build failures might be the same issue as with `f-header` where I had to rebuild the `yarn.lock` file (I deleted the one that was there and ran `yarn` again. There probably a better way to do that..).

**Comment by DamianMullins** [comment]
@dandel10n What happened to the Node v10 build?

**Comment by dandel10n** [comment]
@DamianMullins I found the answer in f-header(to add espree)

**Pull request merged**